src/build.py

```diff
@@ -1190,6 +1198,51 @@ def CompilerRT():
   proc.check_call(['ninja', 'install'], cwd=COMPILER_RT_OUT_DIR, env=cc_env)
 
 
+def LibCXX():
+  buildbot.Step('libcxx')
+  if os.path.isdir(LIBCXX_OUT_DIR):
+    Remove(LIBCXX_OUT_DIR)
+  Mkdir(LIBCXX_OUT_DIR)
+  cc_env = BuildEnv(LIBCXX_SRC_DIR, bin_subdir=True)
+  command = [PREBUILT_CMAKE_BIN, '-G', 'Ninja', os.path.join(LIBCXX_SRC_DIR),
+             '-DCMAKE_CXX_COMPILER_WORKS=ON',
+             '-DCMAKE_C_COMPILER_WORKS=ON',
+             '-DLIBCXX_HAS_PTHREAD_API=ON',
```

Comment:
Is this going to result in a libcxx that requires pthreads on by default? That's probably not what we want, at least for now.

Reply:
You're right.. I found ENABLE_THREADS=OFF.